src/errors.rs

```diff
@@ -0,0 +1,32 @@
+use core::fmt;
+
+#[derive(Debug)]
+pub struct TryIntoError<T> {
```

Comment:
I'd like to add docs for this error type as it's part of public API now.

Reply:
Done in https://github.com/JelteF/derive_more/pull/220/commits/ae3d5c05cee2a8c564edac4d2edc38239319f5f9.